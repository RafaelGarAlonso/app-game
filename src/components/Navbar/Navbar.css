.navbar {
    background-color: #2b3036;
    padding: 10px;
}

.name, .points {
    color: #fff;
    font-size: 16px;
    font-weight: bold;
}