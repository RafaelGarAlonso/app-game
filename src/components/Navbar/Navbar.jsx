import './Navbar.css';

export const Navbar = ({ userName, points }) => {
    return (
        <div className="navbar">
            <div className="name">
                Name: {userName}
            </div>
            <div className="points">
                Points: {points}
            </div>
        </div>
    )
}