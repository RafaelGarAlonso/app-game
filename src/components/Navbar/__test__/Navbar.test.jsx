import { render } from '@testing-library/react';
import { Navbar } from '../Navbar';

describe('Test <Navbar />', () => {

  test('Check snapshot component', () => {
    const { container } = render(<Navbar userName="Manolo" points="100" />);
    expect(container).toMatchSnapshot();
  });

  test('Check that data is rendered', () => {
    const { container } = render(<Navbar userName="Manolo" points="100" />);
    const userName = container.querySelector(".name").innerHTML;
    const points = container.querySelector(".points").innerHTML;

    expect(userName).toBe("Name: Manolo");
    expect(points).toBe("Points: 100");
  });

});