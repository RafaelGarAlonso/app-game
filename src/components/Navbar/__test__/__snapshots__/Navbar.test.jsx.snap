// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Navbar /> Check snapshot component 1`] = `
<div>
  <div
    class="navbar"
  >
    <div
      class="name"
    >
      Name: 
      Manolo
    </div>
    <div
      class="points"
    >
      Points: 
      100
    </div>
  </div>
</div>
`;
